### PR TITLE
Getting SE&L back on track

### PR DESCRIPTION
**Pull request by Bloodmallet**
Stuff wasn't handled very well when we transitioned to Shadowlands. It's time to clean up.

### CODE REVIEW

File: guide/general/covenant.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ By pledging your allegiance to the Necrolords you will gain two new abilites:
 
 **{{site.data.spell.fleshcraft}}**
 
-{{site.data.spell.fleshcraft}} is a channeled ability that gives you a 20% HP shield for 2 minutes. This can be amplified to 50% of your HP if it gets cast while in the vicinity of a corpse.
+{{site.data.spell.fleshcraft}} is a channeled ability that gives you a 40% HP shield for 2 minutes. While chaannelling this reduces damage taken by 20%. Also passively reduces the cooldown by consuming nearby corpses. 1 second per corpse.
```
Comment (HawkCorrigan) [inline guide/general/covenant.md:43]: Typo

File: guide/general/covenant.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
 title: Covenant
-last_update: 2020-12-08 09:00:00
-game_version: 9.0.2 Shadowlands
+last_update: 2021-05-29 09:00:00
+game_version: 9.0.5 Shadowlands
```
Comment (HawkCorrigan) [inline guide/general/covenant.md:5]: Intentional?

File: guide/general/faq.md [hunk 3]
```diff
@@ -44,13 +44,25 @@ Since the cooldown of {{ site.data.spell.meteor }} is the same as the duration o
...
 Not that we're aware of. The Elementals' autocast settings are shared between the Elemental pets. If you deactivate or activate it on one button, the matching button of the other Elementals will share that setting. We hope, that this will be changed.
 
-~~It was changed! 8.3 introduces separate toggle states for all three Elementals and their casts. This now should no longer pose an issue.~~ The fix was reverted just before 8.3 went live.
+~~It was changed! 8.3 introduces separate toggle states for all three Elementals and their casts. This now should no longer pose an issue.~~ The fix was reverted just before 8.3 went live due to bugs introduced from the implementation.
 
 ### Q: "Do I use Lava Burst during Storm Elemental?"
 
 Yes. No. It depends. Check out the [Priority List]({{ site.baseurl }}{% link guide/general/priority_list.md %}) and special notes on that website for in depth information.
 
 ### Q: "What are some useful Macros?"
```
Comment (HawkCorrigan) [inline guide/general/faq.md:53]: Something went wrong here

File: guide/general/legendaries.md [hunk 4]
```diff
@@ -1,17 +1,20 @@
...
 
 ## Legendaries
 
 TLDR:
-Windspeaker for single target
-Skybreakers for spread cleave
-EoGS for M+ and stacked AoE
+{{ site.data.legendary.windspeaker }} for single target
+{{ site.data.legendary.skybreaker }} for M+ and spread cleave
+{{ site.data.legendary.eogs }} for M+ and stacked AoE
```
Comment (HawkCorrigan) [inline guide/general/legendaries.md:14]: Should prolly mention seeds, but it's WIP

File: guide/general/soulbind.md [hunk 5]
```diff
@@ -114,4 +114,4 @@ Dreamweaver while it gives you an extra potency conduit.
 
 Korayn for mythic+ and raids with waves of adds.
 
-Niya for single target.
\ No newline at end of file
+Niya for single target.
```
Comment (HawkCorrigan) [inline guide/general/soulbind.md:117]: Any point talking about Dreamweaver?

File: guide/general/faq.md [hunk 6]
```diff
@@ -44,13 +44,25 @@ Since the cooldown of {{ site.data.spell.meteor }} is the same as the duration o
...
+
```
Comment (HawkCorrigan) [inline guide/general/faq.md:54]: Something went wrong here

File: guide/general/soulbind.md (hunk 5 above)
Comment (Gakamoment) [reply guide/general/soulbind.md]: Dreamweaver is the more common ST build. If anything, Niya is the one that warrants less discussion.

Comment (Bloodmallet) [reply guide/general/soulbind.md]: The soulbind page does no longer talk about any soulbind in specifics. It's just showing simulated values. Feel free to once more add specific content in a separate pull request
